src/AccelStepperFirmata.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (C) 2006-2008 Hans-Christoph Steiner.  All rights reserved.
+  Copyright (C) 2010-2011 Paul Stoffregen.  All rights reserved.
+  Copyright (C) 2009 Shigeru Kobayashi.  All rights reserved.
+  Copyright (C) 2013 Norbert Truchsess. All rights reserved.
+  Copyright (C) 2009-2016 Jeff Hoefs.  All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  See file LICENSE.txt for further informations on licensing terms.
+*/
+
+#ifndef AccelStepperFirmata_h
+#define AccelStepperFirmata_h
+
+#include <ConfigurableFirmata.h>
+#include "utility/AccelStepper.h"
+#include "utility/MultiStepper.h"
+#include "FirmataFeature.h"
+
+#define MAX_STEPPERS 10 // arbitrary value... may need to adjust
```

Comment:
The legacy stepper class declares `MAX_STEPPERS` as 6. In the rare event a user includes both stepper classes there is a potential conflict here.

Reply:
Is it okay to just do this?
````c
#ifndef MAX_STEPPERS
#define MAX_STEPPERS 10
````

Reply:
Nope because if the legacy stepper is included first, that would result in `MAX_STEPPERS` being defined as 6.

Reply:
Imagine defines as global variables in a web application.